Resources/Prototypes/Maps/splitstation.yml

```diff
@@ -6,7 +6,7 @@
     !type:NanotrasenNameGenerator
     prefixCreator: '14'
   mapPath: /Maps/splitstation.yml
-  minPlayers: 40
+  minPlayers: 60
```

Comment:
why 60 now?

Reply:
Split feels a bit empty at 40p imo.

Reply:
60 seems like it would be crowded till we get dynamic and start seeing a handful of people start off station but idc for now. See how it goes.